Article/Article.js

```diff
@@ -3,18 +3,20 @@
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = this.domElement.querySelector('.expandButton');
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    // console.dir(this.expandButton);
+    this.expandButton.innerText = 'expand'
     // Set a click handler on the expandButton reference, calling the expandArticle method.
+    this.expandButton.addEventListener('click', () => this.expandArticle());
```

Comment:
just a quick reminder that this arrow function is used to get around using the .bind(this) method on this function, because arrow functions pass this through the function